WeakAurasOptions/WeakAurasOptions.lua

```diff
@@ -579,6 +579,12 @@ function WeakAuras.ToggleOptions(msg, Private)
         -- The button does not yet exists if a new aura is created
         displayButtons[id]:UpdateWarning()
       end
+      local data = Private.GetDataByUID(uid)
+      for parent in Private.TraverseParents(data) do
+        if displayButtons[parent.id] then
+          displayButtons[parent.id]:UpdateWarning()
+        end
+      end
```

Comment:
You also need to call UpdateWarning() in a few more places, which I think can all be identified by looking for UpdateExpandButton. Though do note that UpdateExpandButton needs to be only called on the direct parent, where as UpdateWarning needs to be called on each parent

Places that seem to be missing:
* CreateNewGroupFromSelection
* Actions["Group"] (That's for drag/drop)
* Actions["Ungroup"]
* In update.lua, which can move auras around.
* OnClickGrouping (which is the grouping icon on the right)